database/dbase.go

```diff
@@ -73,6 +74,7 @@ func ConfigureDbDefault(v cs.Configuration, prefix, envUser, envPasswd string) {
 	v.SetDefault(prefix+"-password", "")
 	v.SetDefault(prefix+"-database", "")
 	v.SetDefault(prefix+"-protocol", "")
+	v.SetDefault(prefix+"-parameters", "")
```

Comment:
Or: v.SetDefault(prefix+"-parameters", "time_zone='UTC'")
... but I don't think it should be the default value